### PR TITLE
Update 02-fundamentals.Rmd

### PR DESCRIPTION
**Pull request by btamsir**
Added additional subsection to detail the demand for data skills.

### CODE REVIEW

**Comment by preranap1** [comment]
I made some cosmetic changes to your file. Check if okay. We can then merge to the contributions branch.

File: 02-fundamentals.Rmd [hunk 1]
```diff
@@ -480,6 +480,14 @@ Here's the process on how to get it done.
 3 | Product Design & Development | Wireframing, prototyping, and application development are the main engines to transform an idea into a final product. Products can range from static presentations/reports to full interactive applications. Mobile, tablet, TV, and workstation platforms can all be mediums to help deliver the final product. The secret to a great end product is how well the data story is conceptualized. If the story is weak then the end product will also suffer. 
 4 | QA & Product Release | The best part of any project is to get it finalized and released for all to see. All data gets verified for accuracy, functionality testing (if applicable), application flow (if applicable), design testing, and remaining items are all completed. The end result is an engaging visual product for all intended audiences to see and use.
 
+### Demand for Data Literacy 
+
+The demand for data literacy is at an all-time high. Originally, data science was focused on the finance and tech industries but the demand for data science skills is increasing for every industrial section. ([Lifeline)](https://lifelinedatacenters.com/data-center/business-intelligence/))
```
Comment (rlwy) [inline 02-fundamentals.Rmd]: I think you got the brackets here mixed up a little. it should be `([Lifeline](https://lifelinedatacenters.com/data-center/business-intelligence/))`

Comment (btamsir) [reply 02-fundamentals.Rmd]: Good Catch. Thanks!

**Comment by rlwy** [comment]
@btamsir i accidentally hit approve but actually you will still need to make one small change. it should be [Lifeline] not [Lifeline)

**Pull request merged**